snapshot-labs/sx-ui
# feat: enable ethTx support

This PR adds adds EthTx authenticator support with support for WalletConnect/Safe.

Depends on https://github.com/snapshot-labs/sx-ui/pull/745

## Test plan
- Connect with WalletConnect to your Safe.
- Go to this space (or create your own with EthTx): http://localhost:8080/#/sn-tn1:0x018e1a59a2ee1d0b1978638be45b9b0153ce7b7799a1c7424776f3dfb0439c86
- Propose or vote.
- Confirm transaction on Safe UI.
- After 10 minutes proposal/vote should show up.
